test/dynamo/test_dynamo.py

```diff
@@ -93,6 +93,49 @@ def test_simple_model(self):
     res_cpu_3 = self.fn_simple(x + y, y * 3)
     self.assertTrue(torch.allclose(res_cpu_3, res_xla_dynamo_3.cpu()))
 
+  # Tests that the dynamo bridge automatically moves tensors to XLA device
+  def test_simple_model_automoves_tensors(self):
+    xla_device = xm.xla_device()
```

Comment:
You can remove this var as it's not used in the test.

Reply:
Done.